deploy/canary/Dockerfile

```diff
@@ -3,8 +3,7 @@ MAINTAINER vmarmol@google.com
 
 RUN apt-get install -y git dmsetup
 RUN git clone https://github.com/google/cadvisor.git /go/src/github.com/google/cadvisor
-RUN go get github.com/tools/godep
-RUN cd /go/src/github.com/google/cadvisor && godep go build .
+RUN cd /go/src/github.com/google/cadvisor && make
```

Comment:
I will clean this up in a separate PR since it needs some work on the jenkins side as well